src/parser-json.js

```diff
@@ -59,9 +61,12 @@ function toBabylon(node) {
         elements: node.children.map(toBabylon)
       });
     case "literal":
-      return Object.assign(result, {
-        rawValue: node.rawValue
-      });
+      return Object.assign(
```

Comment:
I would make a local variable here and extract the type instead of splitting the result into two at the top